Foundation/GTMSQLite.m

```diff
@@ -725,7 +726,8 @@ static void UpperLower16(sqlite3_context *context,
       // we might as well use the preferred encoding of the original call.
       _GTMDevAssert(userArgs->textRep == SQLITE_UTF16BE ||
                     userArgs->textRep == SQLITE_UTF16LE,
-                    @"Received non UTF8 encoding in UpperLower8");
+                    @"Received non UTF16 encoding in UpperLower16, %d",
```

Comment:
nit, you divide your format string with a "," here and a colon above. Standardize?

Reply:
done